.github/workflows/update-icons.yml

```
@@ -0,0 +1,47 @@
+name: Update Icons
+
+on:
+  schedule:
+    - cron: 0 0 * * *
+  workflow_dispatch:
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # Required to retrieve git history
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 14
+      - name: Node-modules cache
+        uses: actions/cache@v2
+        id: node-cache
+        with:
+          path: "**/node_modules"
+          key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
+      - name: install
+        if: steps.node-cache.outputs.cache-hit != 'true'
+        run: yarn install
+      - name: postinstall
+        if: steps.node-cache.outputs.cache-hit == 'true'
+        run: yarn postinstall
+      - name: Clear Tests
+        run: yarn test
+      - name: add Figma token to .env
+        run: echo "FIGMA_TOKEN=${{ secrets.FIGMA_IKONER_API }}" >> @navikt/core/icons/.env
+      - name: Update Icons
+        run: yarn lerna:dl-icons
```

Jeg tror denne kan løses sånn (ikke testet):
```suggestion
      - name: Update Icons
        run: yarn lerna:dl-icons
        env:
          FIGMA_TOKEN: ${{ secrets.FIGMA_IKONER_API }}
```

---

Prøver det, kan hende dotenv blir å klage.